climpred/metrics.py

```diff
@@ -142,20 +229,43 @@ def _crps(forecast, reference, dim='svd', comparison=None):
 
     See also:
         * properscoring.crps_ensemble
+        * xskillscore.crps_ensemble
     """
-    return crps_ensemble(forecast, reference).mean(dim)
+    # switch positions because xskillscore.crps_ensemble(obs, forecasts)
+    return crps_ensemble(reference, forecast)
+
 
+def _crps_gaussian(forecast, mu, sig, **kwargs):
+    """CRPS assuming a guassian distribution. Helper function for CRPSS.
+    See properscoring.crps_gaussian and xskillscore.crps_gaussian."""
+    return crps_gaussian(forecast, mu, sig)
 
-def _crps_gaussian(forecast, mu, sig, dim='svd', comparison=None):
-    return crps_gaussian(forecast, mu, sig).mean(dim)
 
+def _crps_quadrature(forecast, cdf_or_dist, xmin=None, xmax=None, tol=1e-6, **kwargs):
+    """CRPS assuming distribution cdf_or_dist. Helper function for CRPSS.
+    See properscoring.crps_quadrature and xskillscore.crps_quadrature."""
+    return crps_quadrature(forecast, cdf_or_dist, xmin, xmax, tol)
 
-def _crpss(forecast, reference, dim='svd', comparison=None):
+
+def _crpss(forecast, reference, **kwargs):
     """
-    Continuous Ranked Probability Skill Score is strictly proper.
+    Continuous Ranked Probability Skill Score is strictly proper. When assuming
```

Comment:
For standard convention, this should be:

`"""Continuous Ranked Probability Skill Score`

Then below, you could do a `.. note::` that adds all of these other thoughts. Also what does CRPSS is strictly proper mean? Maybe this is obvious and I jsut don't know about the skill score.